lib/pure/os.nim

```diff
@@ -338,8 +344,8 @@ when doslikeFileSystem:
         else:
           return false
 
-proc relativePath*(path, base: string; sep = DirSep): string {.
-  noSideEffect, rtl, extern: "nos$1", raises: [].} =
```

Comment:
Well previously it didn't raise an exception, if it now does, that's a breaking change.

Reply:
I've now added an explanation in changelog documenting this breaking change and in which condition it can now throw